bonus/bonus.js

```diff
@@ -74,13 +95,25 @@ module.exports = { largestNum, oddSum, transpose };
  * @param {number} index - index of array element
  *
  *
+ *
  * @method
  * @name Array#myForEach
  * @param {cb} callback - function to be called with each element
  * @returns {undefined}
  */
-Array.prototype.myForEach = function (callback) {};
+Array.prototype.myForEach = function (callback) {
+  let arr = [];
+  for (let i = 0; i < arr.length; i++) {
+    if (callback(arr[i])) {
```

Comment:
This looks a lot more like how filter works. forEach just calls the callback with each element